### ISSUE TITLE
A Good Idea!

### ISSUE THREAD

**Issue by paul30001** [modrzew/pokeminer#213]
When viewing a report for a single Pokemon:
Say I have 8 Snorlaxes spawned in the past 24 hrs; 4 of them are grouped together on the map, and 4 are in random points. I can see the times they spawned, but I don't know which time belongs to which Snorlax.

@modrzew Is it possible to add an option to view what time a specific instance of a Pokemon spawned? just by clicking on in in the report

**Comment by YonderGod** [comment]
Yes it's possible, reports will be update to use Leaflet soon, so if he doesn't do it then, I will.

**Comment by paul30001** [comment]
@YonderGod 
Yasss, I'll be waiting. Also, how hard do you think it is to add a tool to select an area in /report and then have a box or list come up listing all the Pokemon there are??
This would be very useful, as they are often grouped on top of each other and you can only see the top one.

**Comment by amistad18** [comment]
@paul30001 meanwhile, take a look at this - https://github.com/Cy4n1d3/PokeminerSpawns
